### PR TITLE
FAI-2219: Add `--full` flag + update README

### PR DESCRIPTION
**Pull request by willmarks**
This PR add support for partial events

### CODE REVIEW

**Comment by tovbinm** [comment]
is this a breaking change?

File: README.md [hunk 1]
```diff
@@ -30,40 +30,39 @@ curl -s https://raw.githubusercontent.com/faros-ai/faros-events-cli/$FAROS_CLI_V
...
 An event type (e.g. `CI`, `CD`, `TestExecution`) corresponds to the step of your CI/CD or Test pipeline that you are instrumenting.
 
-- Use Continuous Integration (CI) events to instrument code build pipelines. For example, you can report the result of a successful code build & artifact release:
+- Use Continuous Integration (CI) events to instrument code build pipelines. For example, you can report the result of a successful code build:
 
 ```sh
 ./faros_event.sh CI -k "<faros_api_key>" \
-    --run "<run_source>://<run_organization>/<run_pipeline>/<run_id>" \
     --commit "<commit_source>://<commit_organization>/<commit_repository>/<commit_sha>" \
-    --artifact "<artifact_source>://<artifact_organization>/<artifact_repository>/<artifact_id>" \
```
Comment (willmarks) [inline README.md:52]: Artifact is now more of an advanced configuration option. I am not broadcasting it in the example events.

File: README.md [hunk 2]
```diff
@@ -78,12 +77,61 @@ An event type (e.g. `CI`, `CD`, `TestExecution`) corresponds to the step of your
     --test_suite "<test_suite>" \
     --test_stats "success=5,failure=0,total=5" \
     --test_start_time "1626804356000" \
-    --test_end_time "Now"
+    --test_end_time "Now" \
+    --full
```
Comment (willmarks) [inline README.md:81]: Note: This is the only place where full is currently required. I even left it off of the CI and CD events event when the whole event is provided all at once.

File: README.md [hunk 3]
```diff
@@ -2,71 +2,171 @@
...
+- [Instrumenting CI/CD Pipelines](#instrumenting-cicd-pipelines)
+  - [Reporting Builds with Commits (Basic)](#reporting-builds-with-commits-basic)
+  - [Reporting Deployments with Commits (Basic)](#reporting-deployments-with-commits-basic)
+  - [Reporting Builds & Deployments in Parts (Advanced)](#reporting-builds--deployments-in-parts-advanced)
+  - [Reporting Builds & Deployments with Commits & Artifacts (Advanced)](#reporting-builds--deployments-with-commits--artifacts-advanced)
+- [Code Quality](#code-quality)
+  - [Reporting test execution results](#reporting-test-execution-results)
+- [Usage with Faros Community Edition](#usage-with-faros-community-edition)
+- [Arguments](#arguments)
+- [Tips](#tips)
```
Comment (tovbinm) [inline README.md:16]: 1. Expand `Tips` section here
2. Move `Usage with Faros Community Edition` to `Tips` section
3. `Usage with Faros Community Edition` - remove the example and just mention the necessary flag

File: README.md [hunk 4]
```diff
@@ -2,71 +2,171 @@
...
 
-**Requirements**: Please make sure the following are installed before running the script - `curl`, `jq`, `sed` and an implementation of `awk` (we recommend `gawk`).
+### Option 2: Downloading the Event Script
 
-1. [Download the script manually](https://raw.githubusercontent.com/faros-ai/faros-events-cli/v0.5.2/faros_event.sh) and execute it:
+[Please download the script manually](https://raw.githubusercontent.com/faros-ai/faros-events-cli/v0.5.3/faros_event.sh)
+
+## Quickstart
+
+### Option 1: Execute with Docker
```
Comment (tovbinm) [inline README.md:35]: remove `Option 1` word, and remove `Option 2` word below

**Review by tovbinm — APPROVED** [review]

**Pull request merged**